.github/workflows/CI.yml

```
@@ -272,22 +285,25 @@ jobs:
           target: ${{ matrix.platform.target }}
           args: --release --out dist -i python3.13t
           sccache: ${{ !startsWith(github.ref, 'refs/tags/') }}
           rust-toolchain: nightly
+          working-directory: BinaryOptionsToolsV2
       - name: Upload wheels
         uses: actions/upload-artifact@v4
         with:
           name: wasm-wheels
-          path: dist
+          path: BinaryOptionsToolsV2/dist
       - uses: actions/setup-node@v3
```

actionlint (1.7.10)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)